sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+WITH events_unnested AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id AS client_id,
+    SAFE_CAST(
+      mozfun.norm.browser_version_info(client_info.app_display_version).major_version AS INT64
+    ) AS app_version,
+    normalized_os AS os,
+    normalized_channel AS channel,
+    client_info.locale AS locale,
+    normalized_country_code AS country,
+    metrics.string.newtab_homepage_category AS homepage_category,
+    metrics.string.newtab_newtab_category AS newtab_category,
+    metrics.boolean.pocket_enabled AS organic_content_enabled,
+    metrics.boolean.pocket_sponsored_stories_enabled AS sponsored_content_enabled,
+    metrics.boolean.topsites_sponsored_enabled AS sponsored_topsites_enabled,
+    metrics.boolean.topsites_enabled AS organic_topsites_enabled,
+    metrics.boolean.newtab_search_enabled AS newtab_search_enabled,
+    mozfun.newtab.is_default_ui_v1(
+      category,
+      name,
+      extra,
+      metrics.string.newtab_homepage_category,
+      metrics.string.newtab_newtab_category
+    ) AS is_default_ui,
+    category AS event_category,
+    name AS event_name,
+    extra AS event_details,
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
```

Comment:
```suggestion
    CROSS JOIN UNNEST(events)
```

Reply:
this is not needed. We only care about those with some valid events. Also applied in the [where clause](https://github.com/mozilla/bigquery-etl/blob/e87f31d44fcdb70a6dea3e3d94b51c3c2dbe2819/sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql#L35-L73)

Reply:
The CROSS JOIN returns only not null